tests/selenium/test_selenium_chat_long_message.py

```
@@ -0,0 +1,131 @@
+
+# A ZWJ family emoji + a bidi override span, built from code points so no
+# literal invisible characters live in this source file.
+_ZWJ = chr(0x200D)
+_FAMILY_EMOJI = chr(0x1F468) + _ZWJ + chr(0x1F469) + _ZWJ + chr(0x1F467)
+_BIDI_TEXT = chr(0x202E) + "reversed" + chr(0x202C)
+# Allow a few pixels for sub-pixel rounding / scrollbars.
+_OVERFLOW_TOLERANCE_PX = 5
+
+
+class SeleniumChatLongMessageTest(FHISeleniumBase, StaticLiveServerTestCase):
+    fixtures = ["fighthealthinsurance/fixtures/initial.yaml"]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)